models/Image.php

```diff
@@ -166,27 +202,33 @@ public function ensureThumbnailPath()
     public function generateThumbnail($crop = null)
     {
         ImagineImage::$thumbnailBackgroundAlpha = 0;
-        $size = Yii::$app->params['image.size.thumbnail'];
-        
+        $size                                   = Yii::$app->params['image.size.thumbnail'];
```

Comment:
Just my two cents:

Such an alignment, like here, does not look very pretty, do not you think? I think we need to compile a file describing the code style used, which should be respected by all participants. For example, I was more inclined to use "old" formatting, it is more familiar.